lib/trivial.nix

```diff
@@ -308,7 +306,7 @@ rec {
 
   info = msg: builtins.trace "INFO: ${msg}";
 
-  showWarnings = warnings: res: lib.foldr (w: x: warn w x) res warnings;
+  showWarnings = warnings: res: builtins.foldl' warn res warnings;
```

Comment:
This breaks.

Reply:
I forgot the op for foldl takes a (b -> a -> b) instead of a (a -> b -> b).